google/cloud/storage/testing/storage_integration_test.cc

```diff
@@ -38,6 +38,20 @@ static bool UseGrpcForMedia() {
   return v.find("media") != std::string::npos;
 }
 
+void StorageIntegrationTest::TearDown() {
```

Comment:
https://github.com/google/googletest/blob/master/docs/faq.md#should-i-use-the-constructordestructor-of-the-test-fixture-or-setupteardown-ctorvssetup indicates a preference for `~StorageIntegrationTest()` for cleanup of this variety.  What do you think?

Reply:
Good idea, done.